2996_reflection/reflection.md

```diff
@@ -1660,6 +1744,22 @@ namespace std::meta {
 
 This metafunction produces a reflection representing the constant value of the operand.
 
+### `reflect_invoke`
+
+:::bq
+```c++
+namespace std::meta {
+  consteval auto reflect_invoke(info target, span<info const> args) -> info;
+}
+```
+:::
+
+This metafunction produces a reflection of the value returned by a call expression.
+
+Letting `F` be the entity reflected by `target`, and `A_0, ..., A_n` be the sequence of entities reflected by the values held by `args`: if the expression `F(A_0, ..., A_N)` is a well-formed constant expression evaluating to a type that is not `void`, and if every value in `args` is a reflection of a constant value, then `reflect_invoke(target, args)` evaluates to a reflection of the constant value `F(A_0, ..., A_N)`.
+
+For all other invocations, `reflect_invoke(target, args)` is ill-formed.
```

Comment:
I think we should put `substitute` and `reflect_invoke` next to each other, since they're pretty similar - substitute is `^[: templ :]<[: args :]...>` and `reflect_invoke` is `^[: f :]([: args :]...)`. 

Should also add an example or two.